scipy/stats/tests/test_multivariate.py

```diff
@@ -1226,6 +1237,59 @@ def test_fit_fix_cov(self):
                                                      ).sum()
         assert logp_perturbed < logp_fix
 
+    @pytest.mark.parametrize('X_ndim', [3])
+    @pytest.mark.parametrize('dimensions', [[1], 
+                                            [0, 1]])
```

Comment:
Valid negative indices are not currently testsed, so how about:
```suggestion
                                            [-1, 1]])
```
Also would be fine with me to keep this on the same line.